app/helpers/script_helper.rb

```diff
@@ -21,7 +21,18 @@ def javascript_packs_tag_once(*names, prepend: false)
   end
 
   def render_javascript_pack_once_tags
-    javascript_packs_with_chunks_tag(*@scripts) if @scripts
+    return if !@scripts
+
+    regexp_locale_suffix = %r{\.(#{I18n.available_locales.join('|')})\.js$}
```

Comment:
what if we added a comment like `see getAdditionalAssetFilename` or something to help tell where this pattern comes from?

Reply:
> what if we added a comment like `see getAdditionalAssetFilename` or something to help tell where this pattern comes from?

Added comment in a8f36af.